.github/dependabot.yml

```diff
@@ -19,6 +35,20 @@ updates:
       prefix: "vendor"
     labels:
       - vendor
+    groups:
+      golang-dependencies:
+        patterns:
+          - "github.com/golang*"
+          - "google.golang.org/*"
```

Comment:
include `golang.org/x/*` too?

Reply:
https://github.com/csi-addons/kubernetes-csi-addons/pull/487 here it is